tests/main_test.py

```
@@ -1,189 +1,105 @@
-        author_name_result,
-        author_email_result,
-        commit_signoff_result,
-        merge_base_result,
-        final_result,
-    ):
-        mocker.patch(
-            "commit_check.main.validate_config",
-            return_value={},
-        )
 
+    def test_no_args_shows_help(self, capfd):
```

Ruff (0.13.2)

Unused method argument: `capfd`

(ARG002)